packages/actor-init-sparql/lib/lodash.ts

```diff
@@ -0,0 +1,61 @@
+function mapValues(object: any, func: (key: any, value: any, index: number) => any): void {
+  if (object) {
+    Object.entries(object).forEach(([ key, value ], index) => {
+      object[key] = func(key, value, index);
+    });
+  }
+}
+
+function uniq(array: any[]): any[] {
```

Comment:
This one is used several times, but is quite simple, so I'd just inline it each time.